[2024-08-04T11:07:44.299+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T11:07:44.318+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: export_multiple_dbs_dag.extract_and_save_books scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T11:07:44.327+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: export_multiple_dbs_dag.extract_and_save_books scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T11:07:44.328+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-04T11:07:44.347+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_and_save_books> on 2024-08-03 00:00:00+00:00
[2024-08-04T11:07:44.355+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=95) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-04T11:07:44.356+0000] {standard_task_runner.py:64} INFO - Started process 97 to run task
[2024-08-04T11:07:44.357+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'export_multiple_dbs_dag', 'extract_and_save_books', 'scheduled__2024-08-03T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/export_multiple_dbs_dag.py', '--cfg-path', '/tmp/tmp4o8h5jb8']
[2024-08-04T11:07:44.358+0000] {standard_task_runner.py:91} INFO - Job 148: Subtask extract_and_save_books
[2024-08-04T11:07:44.403+0000] {task_command.py:426} INFO - Running <TaskInstance: export_multiple_dbs_dag.extract_and_save_books scheduled__2024-08-03T00:00:00+00:00 [running]> on host bdbeb3728cf0
[2024-08-04T11:07:44.470+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='export_multiple_dbs_dag' AIRFLOW_CTX_TASK_ID='extract_and_save_books' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-03T00:00:00+00:00'
[2024-08-04T11:07:44.472+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T11:07:44.488+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T11:07:44.488+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T11:07:44.496+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=export_multiple_dbs_dag, task_id=extract_and_save_books, run_id=scheduled__2024-08-03T00:00:00+00:00, execution_date=20240803T000000, start_date=20240804T110744, end_date=20240804T110744
[2024-08-04T11:07:44.531+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-04T11:07:44.558+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-04T11:07:44.561+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
